platform-api/src/services/cf_client.py

```
@@ -0,0 +1,83 @@
+            raise HTTPException(502, f"Cloudflare API error: {resp.status_code}")
+
+        deployments = resp.json().get("result", [])
+        result = deployments[0] if deployments else {}
+
+    # Cache the result in KV
+    if settings.kv and result:
+        await settings.kv.put(cache_key, json.dumps(result), expirationTtl=30)
+        
+    return result
+
+async def get_cf_analytics(metric: str, period: str, settings) -> list:
```

Ruff (0.15.11)

[warning] Unused function argument: `metric`

(ARG001)